_sass/blocks/_revenues_subpage_nav.scss

```diff
@@ -1,35 +1,55 @@
 // Revenues Subpage Nav
 //
-// 
+// This block styles the tags that enable navigation between
+// like subpages
 //
 // Styleguide blocks.revenues_subpage_nav
 
-.revenues-subpage-nav {
-  border-bottom: 1px solid $mid-gray;
-  position: relative;
 
+.revenues_subpage-nav {
+  padding-left: $base-padding;
+  padding-right: $base-padding;
+  @include respond-to(medium-up) {
+    padding-left: 0;
+    padding-right: 0;
+  }
+  
   a {
     color: $light-black;
   }
+}
+.revenues_subpage-nav {
```

Comment:
Merge rule `.revenues_subpage-nav` with rule on line 9<br>Selector `revenues_subpage-nav` should be written in lowercase with hyphens